projects/vdk-plugins/build-plugin.sh

```diff
@@ -30,4 +34,10 @@ fi
 
 pip install pytest-cov
 
-pytest --junitxml=tests.xml --cov vdk --cov-report term-missing --cov-report xml:coverage.xml
+# List exceptions to below check here.
+# Those do not contain any meaningful logic so they do not have any tests defined
```

Comment:
We will have tests in the future perhaps. 

 I prefer to just write some test and not exclude it. You can test the __str__ method :) or just that it can be imported without exception.

Reply:
Ok, I have added a test that checks the __str__ implementation